_Streams/112-happy-holidays.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Happy Holidays from the Coding Train!"
+video_number: 112
+date: 2017-12-21
+video_id: dEWmwQMKgIk
+
+topics:
+  - title: "To skip the stream debugging"
+    time: "22:55"
+  - title: "Snow Coding Challenge"
+    time: "36:32"
+    url: "/CodingChallenges/88-snowfall"
+  - title: "TO SKIP THE SOUND GLITCH AROUND THE 1.50.00 MARK"
+    time: "1:52:43"
+  - title: "Christmas songs!"
+    time: "2:08:11"
+
+links:
+  - title: "Alca's Snowflakes Codepen"
+    url: "https://codepen.io/Alca/pen/ppEgxb"
+  - title: "p5.js reference"
+    url: "https://p5js.org/reference/"
+  - title: "ITP Winter Show 2017"
+    url: "https://itp.nyu.edu/shows/winter2017/show/"
+  - title: "The Coding Train on Amazon"
+    url: "https://www.amazon.com/shop/thecodingtrain"
+
+videos:
```

Comment:
That doesn't make any sense to me. Did you have any particular reason to place those two links under the `video` category?

Reply:
I somehow managed to misplace those links, my bad.